docs/data/joy/components/accordion/accordion.md

```diff
@@ -0,0 +1,58 @@
+---
+product: joy-ui
+title: React Accordion component
+githubLabel: 'component: accordion'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/accordion/
+---
+
+# Accordion
+
+<p class="description">Accordions let users show and hide sections of related content on a page.</p>
+
+:::success
+The Joy Accordion component is still in development, but this should not prevent you from adopting Joy UI today.
+
+This document shows how construct an accordion with existing Joy UI components combined with popular headless UI libraries.
+:::
+
+If you're in need of a Joy UI Accordion, please upvote [this issue](https://github.com/mui/material-ui/issues/36281) to help us prioritize the next batch of new components.
```

Comment:
What do you think about following the same frame as https://mui.com/x/react-data-grid/pivoting/ only so that developer can recognize the pattern.